astropy/io/fits/hdu/base.py

```diff
@@ -526,25 +526,13 @@ def _update_checksum(self, checksum, checksum_keyword='CHECKSUM',
                 del self._header[datasum_keyword]
         elif (modified or self._new or
                 (checksum and ('CHECKSUM' not in self._header or
```

Comment:
It's not really clear to me, but should `'CHECKSUM'` and `'DATASUM'` instead be `checksum_keyword` and `datasum_keyword`?